lib/broccoli/angular-broccoli-less.js

```diff
@@ -0,0 +1,69 @@
+/* jshint node: true, esversion: 6 */
+'use strict';
+
+try {
+  let less;
+
+  if (process.platform === 'win32') {
+    require.resolve(`${process.env.PWD}/node_modules/less`);
```

Comment:
Please just scope the `try {} catch(e) {}` around the `require()`. You'll save a lot of indentation.